spec/controllers/users_controller_spec.rb

```diff
@@ -129,14 +129,14 @@
 
   describe "DELETE #destroy" do
     it "destroys the requested user" do
-      user = User.create! valid_attributes
+      user = FactoryGirl.create :user
       expect {
         delete :destroy, params: {id: user.to_param}, session: valid_session
```

Comment:
É só passar o id do usuário... Filosofia KISS(Keep It Simple Stupid) o "to_param" funciona, mas vamos tentar manter as coisas simples, o o campo espera um id, então passamos um id.